### PR TITLE
chore: add ldflags for version

### PR DESCRIPTION
**Pull request by csatib02**
## Overview

- Adds `ldflags` to explicitly track version.

### CODE REVIEW

**Comment by ramizpolic** [comment]
We should have vars defined in main package to inject the version data into and be able to print them, check this [post](https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications) for details

File: .goreleaser.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ before:
 builds:
   - env:
       - CGO_ENABLED=0
+    ldflags: "-s -w -X main.version={{ .Version }}"
```
Comment (ramizpolic) [inline .goreleaser.yaml:13]:
```suggestion
    ldflags: "-s -w -X main.Version={{ .Version }}"
```

Comment (csatib02) [reply .goreleaser.yaml:13]: ahh yee...

**Review by ramizpolic — APPROVED** [review]

File: .goreleaser.yaml (hunk 1 above)
Comment (ramizpolic) [reply .goreleaser.yaml]: later on, we should make `version` CLI command as well

**Pull request merged**